docs: update screenshots in README

heyanurag opened the pull request:
### Description
Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes #758 

### Type of Change:
**Delete irrelevant options.**

- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)



### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.


### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] I have written Kotlin Docs whenever is applicable


**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

heyanurag committed:
docs: update new screenshots in README

heyanurag committed:
Update README.md

docs: update screenshots in README

singhanurag05 commented:
I needed some help with the markdown, why are the images in the first column bigger than the other images? I tried to correct it several times so thought of creating a PR and asking the doubt.

heyanurag committed:
docs: update screenshots in README

heyanurag commented:
the changes are visible [here](https://github.com/singhanurag05/mentorship-android/tree/update-screenshots)
@vj-codes @aditmehta9 @sanchi0204 When will the PR be reviewed?

singhanurag05 commented:
@anna4j @sanchi0204 Thanks for reviewing! May I know why did this PR not reflect on my hacktoberfest page? It still says pending

isabelcosta reviewed (changes requested):
@singhanurag05 could you please upload these images in a comment and then use instead the link of the URLs on the README? In this way, we can remove the binary files from the repository. Having these binary files adds unnecessary weight to this repository.

Here's a guide if you have any doubt on how to upload the images: https://gist.github.com/vinkla/dca76249ba6b73c5dd66a4e986df4c8d

heyanurag commented:
I'll be doing this on the 7th of November as I have my mid semester exams starting tomorrow. Would that be fine?

sanchi0204 commented:
> I'll be doing this on the 7th of November as I have my mid semester exams starting tomorrow. Would that be fine?
@singhanurag05 pls update this PR as per the changes requested

isabelcosta commented:
@singhanurag05 are you still going to work on this issue? It is ok if not, we just need to know for triaging issues and pull requests purposes

heyanurag commented:
I currently wont be able to complete this. Going through a rough patch in life. really sorry for holding stuff up for this long

aditmehta9 commented:
@singhanurag05 can you update the PR?

heyanurag commented:
yes sure!

heyanurag commented:
![About Page](https://user-images.githubusercontent.com/63957920/107955159-76f4e980-6fc3-11eb-87f8-464d6e9b9db4.jpeg)
![Change Password Page](https://user-images.githubusercontent.com/63957920/107955177-7bb99d80-6fc3-11eb-8a61-729001a3c330.jpeg)
![Edit Profile Page](https://user-images.githubusercontent.com/63957920/107955188-7f4d2480-6fc3-11eb-8698-66f41d6ae835.jpeg)
![Filter Members Page](https://user-images.githubusercontent.com/63957920/107955194-81af7e80-6fc3-11eb-8ce7-c08d7e39b22a.jpeg)
![Home Page](https://user-images.githubusercontent.com/63957920/107955205-84aa6f00-6fc3-11eb-95c7-4db92f18f6df.jpeg)
![Introduction Slider](https://user-images.githubusercontent.com/63957920/107955208-86743280-6fc3-11eb-9d5f-0e1031beb992.jpeg)
![Login Page](https://user-images.githubusercontent.com/63957920/107955213-883df600-6fc3-11eb-9d43-768a25f11aff.jpeg)
![Member Profile Page](https://user-images.githubusercontent.com/63957920/107955215-8a07b980-6fc3-11eb-90b8-cde542c517bd.jpeg)
![Members Page](https://user-images.githubusercontent.com/63957920/107955249-9429b800-6fc3-11eb-9a67-9a9cdafcb8ba.jpeg)
![Profile Page](https://user-images.githubusercontent.com/63957920/107955253-94c24e80-6fc3-11eb-9463-39aa50cb382c.jpeg)
![Relations Page](https://user-images.githubusercontent.com/63957920/107955255-955ae500-6fc3-11eb-8d00-2664a2dce0cc.jpeg)
![Requests Page](https://user-images.githubusercontent.com/63957920/107955257-955ae500-6fc3-11eb-8aec-488ed1cd8e0a.jpeg)
![Send Request Page](https://user-images.githubusercontent.com/63957920/107955259-95f37b80-6fc3-11eb-9e4f-5a284d82195a.jpeg)
![Settings Page](https://user-images.githubusercontent.com/63957920/107955260-95f37b80-6fc3-11eb-8862-4e0668c6d6e1.jpeg)
![SignUp Screen](https://user-images.githubusercontent.com/63957920/107955263-968c1200-6fc3-11eb-952d-def226ef1174.jpeg)
![Splashscreen](https://user-images.githubusercontent.com/63957920/107955265-968c1200-6fc3-11eb-9651-b85fc5c670c8.png)
![Tasks Page](https://user-images.githubusercontent.com/63957920/107955267-9724a880-6fc3-11eb-9b20-c11ee5013014.jpeg)

heyanurag committed:
Update README.md

uploaded photos in a github comment (https://github.com/anitab-org/mentorship-android/pull/1017#issuecomment-779239560) and added the image urls in the readme so that the screenshots folder could be removed from the repo.

heyanurag commented:
@aditmehta9 @isabelcosta I have made the changes please check.

aditmehta9 commented on README.md:
```diff
@@ -63,12 +63,13 @@ The repository has the following permanent branches:
+|:---:|:---:|:---:|
+|![Home Page](https://user-images.githubusercontent.com/63957920/107955205-84aa6f00-6fc3-11eb-95c7-4db92f18f6df.jpeg)|![Profile Page](https://user-images.githubusercontent.com/63957920/107955253-94c24e80-6fc3-11eb-9463-39aa50cb382c.jpeg)|![Edit Profile Page](https://user-images.githubusercontent.com/63957920/107955188-7f4d2480-6fc3-11eb-8698-66f41d6ae835.jpeg)|
+|![Relations Page](https://user-images.githubusercontent.com/63957920/107955255-955ae500-6fc3-11eb-8d00-2664a2dce0cc.jpeg)|![Tasks Page](https://user-images.githubusercontent.com/63957920/107955267-9724a880-6fc3-11eb-9b20-c11ee5013014.jpeg)|![Members Page](https://user-images.githubusercontent.com/63957920/107955249-9429b800-6fc3-11eb-9a67-9a9cdafcb8ba.jpeg)|
+|![Filter Mumbers Screen](https://user-images.githubusercontent.com/63957920/107955194-81af7e80-6fc3-11eb-8ce7-c08d7e39b22a.jpeg)|![Member Profile](https://user-images.githubusercontent.com/63957920/107955215-8a07b980-6fc3-11eb-90b8-cde542c517bd.jpeg)|![Send Request Page](https://user-images.githubusercontent.com/63957920/107955259-95f37b80-6fc3-11eb-9e4f-5a284d82195a.jpeg)|
```
Can you change the name from Filter Mumbers to Filter Members

heyanurag committed:
Update README.md

aditmehta9 commented:
@vj-codes can you review this?

aditmehta9 reviewed (dismissed):
LGTM

vj-codes commented:
> @singhanurag05 could you please upload these images in a comment and then use instead the link of the URLs on the README? In this way, we can remove the binary files from the repository. Having these binary files adds unnecessary weight to this repository.
> 
> Here's a guide if you have any doubt on how to upload the images: https://gist.github.com/vinkla/dca76249ba6b73c5dd66a4e986df4c8d

@singhanurag05 hey almost done but as mentioned above you will need to remove the binary files as well from the PR.
This will also lead to removing the Screenshots folder from the repo as they are unnecessary weightage please update acc to this and so it'll just be one file changed (reamde) instead of 31 files in your PR and squash your commits if possible.
That's it and then it's good to merge, @aditmehta9 agree?

aditmehta9 commented:
> 
> 
> > @singhanurag05 could you please upload these images in a comment and then use instead the link of the URLs on the README? In this way, we can remove the binary files from the repository. Having these binary files adds unnecessary weight to this repository.
> > Here's a guide if you have any doubt on how to upload the images: https://gist.github.com/vinkla/dca76249ba6b73c5dd66a4e986df4c8d
> 
> @singhanurag05 hey almost done but as mentioned above you will need to remove the binary files as well from the PR.
> This will also lead to removing the Screenshots folder from the repo as they are unnecessary weightage please update acc to this and so it'll just be one file changed (reamde) instead of 31 files in your PR and squash your commits if possible.
> That's it and then it's good to merge, @aditmehta9 agree?

Ohh yes I agree I didn't notice that.

heyanurag committed:
Delete Screenshots directory

heyanurag commented:
i cant figure out how to squash commits but I think someone who is able to merge the pr can squash the commits

vj-codes commented:
@aditmehta9 can you review this again please?

aditmehta9 commented:
> 
> 
> @aditmehta9 can you review this again please?

Yes, sure, I just noticed that there isn't splash screen screenshot in this PR is it ok? Should I approve this PR?

vj-codes commented:
@aditmehta9 there's no splash screen currently for the apk

aditmehta9 commented:
> 
> 
> @aditmehta9 there's no splash screen currently for the apk

Ok thanks

vj-codes reviewed (approved):
LGTM 🎉 
@singhanurag05 thank you for making the changes:)

vj-codes commented:
Adding label ready to merge since it's documentation change

aditmehta9 commented:
Tested after changes:

https://user-images.githubusercontent.com/35133994/108126831-5070a480-70d0-11eb-956a-3746301efe00.mp4

heyanurag commented:
i have changed my username to @heyanurag so please tag me there if there are further changes
